### DISCUSSION TITLE
Hello there

### DISCUSSION THREAD

**Discussion by kevinstiller** [nkasco/IT-Admin-Toolkit-WinUI#47] [General]
Is this still usable? I have tested this, but I can't start the application, although I have installed the correct version of Windows App Runtime. Tested it with the current version and 1.2 and 1.2.2. :(

**Comment by nkasco** [comment]
Take a look at these:
https://github.com/nkasco/IT-Admin-Toolkit-WinUI/issues/29#issuecomment-1378170698
https://github.com/nkasco/IT-Admin-Toolkit-WinUI/issues/44#issuecomment-1454914579
